src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/pullrequest/BitbucketPullRequestValue.java

```diff
@@ -93,6 +99,15 @@ public String getAuthorDisplayName() {
         return author.displayName;
     }
 
+    @Override
+    public List<BitbucketReviewerValue> getReviewers() {
```

Comment:
You don't seem to need the `BitbucketReviewer` interface here?

Reply:
`BitbucketPullRequestFull`'s contract is that it will return a list of something that extends from `BitbucketReviewer` (`List<? extends BitbucketReviewer>`) and thus here we are returning a list of `BitbucketReviewerValue` and in `BitbucketServerPullRequest` we are returning a list of `BitbucketServerReviewer`. So while we aren't using `BitbucketReviewer` explicitly here we are implementing an interface that uses it. 
Hopefully I didn't misinterpret your question there.